src/Extractor/OdbcExtractor.php

```diff
@@ -49,10 +50,13 @@ protected function createConnection(DatabaseConfig $dbConfig): void
     protected function createExportAdapter(): ExportAdapter
     {
         $queryFactory = new OdbcQueryFactory($this->state);
+        $resultWriter = new DefaultResultWriter($this->state);
+        $resultWriter->setIgnoreInvalidUtf8();
```

Comment:
A tu je druhy problem ... ani po fixnuti CLIENT LOCALE do UTF-8, to klientovi nejde. Zistili sme, ze Informix je velmi tolerantny, a uzmonil ulozit do DB aj neplatne UTF-8 znaky. Takze jedine co vieme spravit je ich ignorovat. 
Problem je mozno v jednom riadku z 1M, no blokuje to export.

Reply:
... pri dalsom skumani som zistil, ze asi ide skor o bug v ODBC drivery, a data su v db asi dobre ... v kazdom pripade, s tym nevieme teraz viac spravit ako tieto zle znaky ignorovat.